### ISSUE TITLE
Saw a non-null label (index >= num_classes - 1) following a null label 

### ISSUE THREAD

**Issue by lxj0276** [ypwhs/captcha_break#7]
现在运行ctc'代码时出现这个问题，我想问下你当初的tensorflow的版本是哪个

**Comment by ypwhs** [comment]
忘记了，你可以试试更新到最新版，应该是可以跑的。

**Comment by GabrielLin** [comment]
The same error shew at Keras 2.0.5 and Tensorflow 1.3.0.

**Comment by Sqrt5** [comment]
尝试把n_class+1程序可以运行，但是不知道对不对
x = Dense(n_class + 1, kernel_initializer='he_normal', activation='softmax')(x)

**Comment by sysustrange** [comment]
同样有这个问题，在使用了 @Sqrt5 的方法之后得到解决，希望可以求证一下可行性。

**Comment by ypwhs** [comment]
现在使用 tensorflow 1.13.1 测试完成，模型训练速度快了，精度也高了，你可以试试最新的代码。
